### PR TITLE
Translation: update Korean translation

### PR DESCRIPTION
**Pull request by potatogim**
Hello!

Thanks for letting me use Joplin! :) 

I updated some translation for Korean.

Signed-off-by: Ji-Hyeon Gim <potatogim@gluesys.com>

### CODE REVIEW

File: CliClient/locales/ko.po [hunk 1]
```diff
@@ -3201,105 +3208,3 @@ msgstr "OneDrive 로그인"
...
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "파일 시스템 동기화가 활성화된 경우에 동기화를 할 경로입니다. `sync.target`"
-#~ "를 참조하세요."
-
-#~ msgid "State: %s."
-#~ msgstr "상태: %s."
-
-#~ msgid "A notebook with this title already exists: \"%s\""
-#~ msgstr "같은 제목의 노트북이 이미 있습니다: \"%s\""
```
Comment (tessus) [inline CliClient/locales/ko.po]: Do not remove these old translations.

File: CliClient/locales/ko.po [hunk 2]
```diff
@@ -1708,12 +1710,13 @@ msgstr[0] "%s 새로 만들기..."
 #, fuzzy
 msgid "Link has been copied to clipboard!"
 msgid_plural "Links have been copied to clipboard!"
-msgstr[0] "토큰이 클립보드로 복사되었습니다!"
+msgstr[0] "링크가 클립보드로 복사되었습니다!"
```
Comment (tessus) [inline CliClient/locales/ko.po]: this is still marked as fuzzy

File: CliClient/locales/ko.po [hunk 3]
```diff
@@ -1724,11 +1727,11 @@ msgstr "복구된 노트들"
 #, fuzzy
 msgid "Copy Shareable Link"
 msgid_plural "Copy Shareable Links"
-msgstr[0] "공유"
+msgstr[0] "공유 링크 복사"
```
Comment (tessus) [inline CliClient/locales/ko.po]: still marked as fuzzy

File: CliClient/locales/ko.po (hunk 1 above)
Comment (potatogim) [reply CliClient/locales/ko.po]: Yep! I have pushed new commit to restore that!

File: CliClient/locales/ko.po (hunk 2 above)
Comment (potatogim) [reply CliClient/locales/ko.po]: Thanks! I removed fuzzy :)

File: CliClient/locales/ko.po (hunk 3 above)
Comment (potatogim) [reply CliClient/locales/ko.po]: Thanks! I removed fuzzy :)

**Comment by potatogim** [comment]
@tessus I pushed new commit that includes more changes for fuzzy messages.

**Pull request merged**